src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1385,9 +1399,16 @@ class ContentExplorer extends Component<Props, State> {
                 event.preventDefault();
                 break;
             case 'arrowdown':
-                focus(this.rootElement, '.bce-item-row', false);
-                this.setState({ focusedRow: 0 });
-                event.preventDefault();
+                if (viewMode === VIEW_MODE_GRID) {
+                    if (!this.isFocusedOnItem()) {
+                        focus(this.rootElement, '.btn-plain.be-item-label', false);
+                        event.preventDefault();
+                    }
+                } else {
+                    focus(this.rootElement, '.bce-item-row', false);
+                    this.setState({ focusedRow: 0 });
+                    event.preventDefault();
+                }
```

Comment:
if we are in grid mode, then we want to focus the first item in the grid, and allow keyboard navigation. Once we are in the grid, then hitting arrowDown will be handled by the ArrowKeyStepper